test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerIntegrationTest.cs

```diff
@@ -143,7 +143,9 @@ public async Task Build_ServerConnectionMutexCreationFails_FallsBackToInProcessR
             // it reaches server creation part.
         }
 
-        [Fact]
+        // Skipping on linux/mac because of https://github.com/aspnet/Razor/issues/2507.
```

Comment:
The problem is not that we can't shutdown the sever. It is because this test causes the fixture server to shutdown prematurely before running all the other tests here. Nevertheless, I still want to figure out the exact root cause. I've updated the [issue](https://github.com/aspnet/Razor/issues/2525) to reflect the priority.